app/src/main/java/org/oppia/android/app/onboarding/OnBoardingViewPagerViewModel.kt

```diff
@@ -0,0 +1,5 @@
+package org.oppia.android.app.onboarding
+
+import org.oppia.android.app.viewmodel.ObservableViewModel
+
+abstract class OnBoardingViewPagerViewModel : ObservableViewModel()
```

Comment:
Nit here & elsewhere: 'onboarding' is one word so it should be 'onboarding' or 'Onboarding' (e.g. the 'b' is never capitalized).

Reply:
Done.